marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/WriteStepRunner.java

```diff
@@ -306,6 +306,15 @@ public RunStepResponse run() {
             if(fileLocations.get("outputURIReplacement") != null) {
                 this.outputURIReplacement = fileLocations.get("outputURIReplacement");
             }
+            if(fileLocations.get("separator") != null) {
+                if(! this.inputFileType.equalsIgnoreCase("csv")){
+                    throw new IllegalArgumentException("Invalid argument for file type " + inputFileType );
```

Comment:
I'd add a little more context to this - i.e. "When specifying a separator, the file type must be 'csv'"